src/zcl_mustache_parser.clas.testclasses.abap

```diff
@@ -105,14 +93,25 @@ class ltcl_mustache_parser implementation.
     append '* {{name}} - ${{price}}' to lt_template.
     append '{{/items}}'              to lt_template.
 
-    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  1   `Our sales:`.
-    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  1   c_nl.
-    _add_mu_token lt_exp zif_mustache=>c_token_type-section     '=' 1   'items'.
-    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  2   `* `.
-    _add_mu_token lt_exp zif_mustache=>c_token_type-etag        ''  2   'name'.
-    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  2   ` - $`.
-    _add_mu_token lt_exp zif_mustache=>c_token_type-etag        ''  2   'price'.
-    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  2   c_nl.
+    lt_exp = VALUE #(
+      ( type = zif_mustache=>c_token_type-static cond = '' level = 1 content = `Our sales:` )
+      ( type = zif_mustache=>c_token_type-static cond = '' level = 1 content = c_nl )
+      ( type = zif_mustache=>c_token_type-section cond = '=' level = 1 content = `items` )
+      ( type = zif_mustache=>c_token_type-static cond = '' level = 2 content = `* ` )
+      ( type = zif_mustache=>c_token_type-etag cond = '' level = 2 content = `name` )
+      ( type = zif_mustache=>c_token_type-static cond = '' level = 2 content = ` - $` )
+      ( type = zif_mustache=>c_token_type-etag cond = '' level = 2 content = `price` )
+      ( type = zif_mustache=>c_token_type-static cond = '' level = 2 content = c_nl )
+    ).
+
+*    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  1   `Our sales:`.
+*    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  1   c_nl.
+*    _add_mu_token lt_exp zif_mustache=>c_token_type-section     '=' 1   'items'.
+*    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  2   `* `.
+*    _add_mu_token lt_exp zif_mustache=>c_token_type-etag        ''  2   'name'.
+*    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  2   ` - $`.
+*    _add_mu_token lt_exp zif_mustache=>c_token_type-etag        ''  2   'price'.
+*    _add_mu_token lt_exp zif_mustache=>c_token_type-static      ''  2   c_nl.
```

Comment:
Could you remove the comments pls ?

Reply:
Comment is removed, sorry, was a leftover